relayer/middleware/wallet/walletToolBox.ts

```diff
@@ -94,3 +100,25 @@ function createSuiWalletToolBox(
     address: address,
   };
 }
+
+async function createSeiWalletToolBox(
+  providers: Providers,
+  mnemonic: string,
+): Promise<WalletToolBox<SeiWallet>> {
+  const seiWallet = await DirectSecp256k1HdWallet.fromMnemonic(mnemonic, {
+    prefix: "sei",
+  });
+  const [seiAccount] = await seiWallet.getAccounts();
+
+  const seiProvider = providers.sei[0];
+
+  return Promise.resolve({
```

Comment:
Why are you using `Promise.resolve` in this case? :thinking:

Reply:
It's an `async` function that returns a promise. So we want to return a resolved promise with the `WalletToolBox<SeiWallet>` object.

Reply:
it's not really necessary to do so. Just returning the object will still return a promise because of the `async` keyword preceding the function